modules/executive/components/VoteModal/DefaultView.tsx

```diff
@@ -123,7 +123,8 @@ export default function DefaultVoteModalView({
         // if comment included, add to comments db
         if (comment.length > 0) {
           const requestBody: ExecutiveCommentsRequestBody = {
-            voterAddress: account || '',
+            voterAddress: addressLockedMKR || '',
+            hotAddress: account || '',
```

Comment:
in the case of a plain user who isn't a delegate or VP, `hotAddress` and `voterAddress` will be the same, right?

Reply:
yes